wp-content/themes/pub/wporg-learn-2024/src/sensei-meta-list/index.php

```diff
@@ -90,6 +90,48 @@ function render( $attributes, $content, $block ) {
 				'key'   => 'last-updated',
 			),
 		);
+	} else if ( 'lesson-plan' === $block->context['postType'] ) {
+		$lesson_plan_id = $block->context['postId'];
+
+		$duration = get_post_taxonomy_terms_links( $lesson_plan_id, 'duration' );
+		$audience = get_post_taxonomy_terms_links( $lesson_plan_id, 'audience' );
+		$level = get_post_taxonomy_terms_links( $lesson_plan_id, 'level' );
+		$instruction_type = get_post_taxonomy_terms_links( $lesson_plan_id, 'instruction_type' );
+		$wporg_wp_version = get_post_taxonomy_terms_links( $lesson_plan_id, 'wporg_wp_version' );
```

Comment:
These links aren't the same as the existing links.

Current theme link (for version 5.8): https://learn.wordpress.org/lesson-plans/?wp_version[]=3524
New theme link (for same version): https://learn.wordpress.org/wporg_wp_version/5-8/

The new theme link takes me to the homepage; it doesn't seem to resolve. Ideally it would take you to the Lesson Plan archive with the version filter applied, but as we don't have this and version isn't something that's being used anywhere else, I think we should remove the link and possibly even the field itself.

Reply:
Yeah, exactly. I directed the link to the term archive page here because there is no filter function and apart from the "wp version", all other terms are accessible. It appears that "wp version" was initially set to not allow viewing the archive page: ```'public' => false.``` If we set it to true, then its archive page can be accessed. It seems fine to allow it should be fine. What do you think?

![image](https://github.com/WordPress/Learn/assets/18050944/b810aa46-7ae5-4809-b66e-2671fed05d69)

Reply:
**Update**

Hold on, I see your comment below, yeah, this field should either be unlinked or just removed, as this term archives not only filtering Lesson Plans.

Reply:
**Update**
When I set:
```
'public'            => true,
'rewrite'           => false,
```
it achieves the following effect - you can see all lesson plan posts of version 6.1, but it won't reflect in the filter.

![image](https://github.com/WordPress/Learn/assets/18050944/e07957c5-109b-4589-b500-b566c9403356)

Do you think this setting is good to make the version archive accessible? Do we want to add a specific filter? 
If we add a filter for duration, it will only be used for lesson plans. The version will be additionally used for tutorials.

Or we just don't link any of them at all as you said and just present it as additional information to the user, especially since it's going to be deprecated. If we do decide to link, adding a filter seems to make more sense.